docs/ARCHITECTURE.md

```diff
@@ -78,7 +78,7 @@ events received from Astarte (based on the declared Introspection) will be redir
 
 ```protobuf
 service MessageHub {
-  rpc Attach(Node) returns (stream AstarteMessage) {}
+  rpc Attach(Node) returns (stream MessageHubEvent) {}
   ....
 }
 ```
```

Comment:
Remove the node_id in Message `Node` https://github.com/astarte-platform/astarte-message-hub/blob/f949777e90466b44999ff294462de68235b60820/docs/ARCHITECTURE.md?plain=1#L68 from this file